core/templates/dev/head/components/share/sharing_collection_links_directive.html

```diff
@@ -0,0 +1,74 @@
+<script type="text/ng-template" id="components/sharingCollectionLinks">
```

Comment:
How many logic choices are there? I only see "show embed button?" and "current URL". Is there anything else?

If not, let's create a single component, so that we only have to update one place each time. Otherwise, maybe you could say what the other choices are, and we can continue discussing.